dist/angular-patternfly.js

```diff
@@ -302,7 +304,7 @@ angular.module('patternfly.card').directive('pfCard', function () {
       $scope.updateAvailable();
       $scope.chartConfig.data.columns = [["Used",$scope.used],["Available",$scope.available]];
    };
- }
+  }]);
```

Comment:
Nitpick:  these closing brackets and the parenthesis don't line up with the `angular.module` above.  This is also visible in the docs examples.